challenges/2022-06-14-two-sum-numbers/solutions/javascript/Moses-Alero/TwosumNumbers.js

```diff
@@ -1,9 +1,8 @@
 const TwoSumNumbers = function (number, numbersArray){
 
-   for(let i = 0; i < numbersArray.length;i++){
+   while(numbersArray.length > 0){
         const summation = number - numbersArray[0];
         numbersArray.splice(0,1);
```

Comment:
This is the same as `numbersArray.shift()`. May as well use the shift method.

Reply:
Thanks for your input will get to that now